.circleci/config.yml

```diff
@@ -179,10 +179,10 @@ jobs:
       - run:
           name: Download additional WP Plugins for tests
           command: |
-            ./do download:woo-commerce-zip 6.2.0
-            ./do download:woo-commerce-subscriptions-zip 3.0.14
-            ./do download:woo-commerce-memberships-zip 1.22.9
-            ./do download:woo-commerce-blocks-zip 7.2.1
+            ./do download:woo-commerce-zip 6.6.0
```

Comment:
I think that we should use L-2 versions only for the oldest tests. Here we should use the latest stable release (L).
So, e.g. WooCommerce should be fixed to 6.8.0